### PR TITLE
ENG-11839: Updates the download instructions for the demo notebooks

### PR DESCRIPTION
**Pull request by bredamc**
ENG-11839: Updates the download instructions for the demo notebooks

### CODE REVIEW

File: modules/running-distributed-data-science-workloads-from-notebooks.adoc [hunk 1]
```diff
@@ -48,110 +51,164 @@ If your cluster administrator does not define a default local queue, you must sp
...
+* You have downloaded the demo notebooks provided by the CodeFlare SDK, as described in link:{odhdocshome}/working_with_distributed_workloads/#downloading-the-demo-notebooks-from-the-codeflare-sdk_distributed-workloads[Downloading the demo notebooks from the CodeFlare SDK].
+endif::[]
+
+
 .Procedure
-. Download the demo notebooks provided by the CodeFlare SDK.
-The demo notebooks provide guidelines for how to use the CodeFlare stack in your own notebooks.
+. In the JupyterLab interface, open the *demo-notebooks > guided-demos* folder. 
+. Double-click each notebook in turn to open the notebook.
+Notebook files are named `_<filename>_.ipynb`.
```
Comment (eturner24) [inline modules/running-distributed-data-science-workloads-from-notebooks.adoc:87]: Why should they take note of the filename? I only see one other notebook name called out - do we need this?

File: assemblies/running-distributed-workloads.adoc [hunk 2]
```diff
@@ -8,8 +8,9 @@ ifdef::context[:parent-context: {context}]
 [role='_abstract']
 
 In {productname-short}, you can run a distributed workload from a notebook or from a pipeline.
-You can also run distributed workloads in a disconnected environment if you have access to all of the required software.
+You can run distributed workloads in a disconnected environment if you can access all of the required software (including a Ray cluster image, and the data sets and Python dependencies used by the workload) from the disconnected environment.
```
[the remark this answers is not in the document]
Comment (bredamc) [reply assemblies/running-distributed-workloads.adoc]: Updated the text to clarify.

File: modules/downloading-the-demo-notebooks-from-the-codeflare-sdk.adoc [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+** If you did not create the project, your cluster administrator must give you Admin access.
+
+* You have logged in to {productname-long}.
+* You have launched your notebook server and logged in to your notebook editor.
+The examples in this procedure refer to the JupyterLab integrated development environment (IDE).
+
+.Procedure
+. In the JupyterLab interface, click *File > New > Notebook*, and then click *Select*. 
++
+A new notebook is created in a file with a default name, for example, `Untitled.ipynb`.
```
[the remark this answers is not in the document]
Comment (bredamc) [reply modules/downloading-the-demo-notebooks-from-the-codeflare-sdk.adoc]: Updated the text to clarify.

File: modules/running-distributed-data-science-workloads-from-notebooks.adoc [hunk 4]
```diff
@@ -48,110 +51,164 @@ If your cluster administrator does not define a default local queue, you must sp
...
+ifdef::upstream[]
+* You have downloaded the demo notebooks provided by the CodeFlare SDK, as described in link:{odhdocshome}/working_with_distributed_workloads/#downloading-the-demo-notebooks-from-the-codeflare-sdk_distributed-workloads[Downloading the demo notebooks from the CodeFlare SDK].
+endif::[]
+
+
 .Procedure
-. Download the demo notebooks provided by the CodeFlare SDK.
-The demo notebooks provide guidelines for how to use the CodeFlare stack in your own notebooks.
+. In the JupyterLab interface, open the *demo-notebooks > guided-demos* folder. 
+. Double-click each notebook in turn to open the notebook.
```
[the remark this answers is not in the document]
Comment (bredamc) [reply modules/running-distributed-data-science-workloads-from-notebooks.adoc]: Updated the text to clarify.

File: modules/running-distributed-data-science-workloads-from-notebooks.adoc (hunk 1 above)
Comment (bredamc) [reply modules/running-distributed-data-science-workloads-from-notebooks.adoc]: Updated the text to clarify.

File: modules/running-distributed-data-science-workloads-from-notebooks.adoc [hunk 5]
```diff
@@ -48,110 +51,164 @@ If your cluster administrator does not define a default local queue, you must sp
...
+----
+from codeflare_sdk import Cluster, ClusterConfiguration, TokenAuthentication
+----
 
-* `overwrite: bool = False` specifies whether the function should overwrite the contents of an existing directory.
-When this argument is set to `True`, the function overwrites your customizations in the original demo files.
-The default value is `False`.
+. In each notebook, update the `TokenAuthentication` section to provide the `token` and `server` details to authenticate to the OpenShift cluster by using the CodeFlare SDK.
++
+You can identify your token and server details as follows:
```
[the remark this answers is not in the document]
Comment (bredamc) [reply modules/running-distributed-data-science-workloads-from-notebooks.adoc]: Replaced with the simpler "find" :)

**Comment by bredamc** [comment]
@eturner24 Many thanks for your great review comments! I've updated the text accordingly (albeit not always exactly as you suggested!). 

Would you like to review again, or are you happy for me to merge now?

**Review by eturner24 — APPROVED** [review]
These revisions look great! Thank you, Breda!

**Pull request merged**